src/adapters/convex/products/cvxcrv-wrapper/convexCvxcrvWrapperAdapter.ts

```diff
@@ -61,10 +61,19 @@ export class ConvexCvxcrvWrapperAdapter
   async getPositions({
     userAddress,
     blockNumber,
+    protocolTokenAddresses,
   }: GetPositionsInput): Promise<ProtocolPosition[]> {
-    const [protocolTokens] = await this.getProtocolTokens()
+    const [protocolToken] = await this.getProtocolTokens()
```

Comment:
i think your idea about passing the protocolTokenAddresses to this.getProtocolTokens() was a nice idea. Anyway lets do it later